Session/Conversations/ConversationVC+Interaction.swift

```diff
@@ -351,7 +357,16 @@ extension ConversationVC : InputViewDelegate, MessageCellDelegate, ContextMenuAc
                     }
                 )
             }
-        )
+        
+            // Show an error indicating that approving the thread failed
+            promise.catch(on: DispatchQueue.main) { [weak self] _ in
+                let alert = UIAlertController(title: "Session", message: "An error occurred when trying to accept this message request", preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
```

Comment:
Could you add the error message to translation files? Cheers!

Reply:
Ah yep, sorry meant to do this but fell off my list